tests/encapsulate_derived.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -xeuo pipefail
+# Pull the latest FCOS build, unpack its container image, and verify
+# that we can re-encapsulate it as chunked.
+
+ver=37.20230201.20.0
+packaging_ver=37.20221111.20.0
+compare_ver=37.20230131.20.0
+container=quay.io/fedora/fedora-coreos:${ver}
+
+tmpdir=/tmp/coreTest-${ver}
+p_tmpdir=/tmp/coreTest-${packaging_ver}
+mkdir -p ${tmpdir}
+cd ${tmpdir}
+ostree --repo=repo init
+cat /etc/ostree/remotes.d/fedora.conf >> repo/config
+
+# Pull and unpack the ostree content, discarding the container wrapping
+ostree container unencapsulate --write-ref=testref --repo=repo ostree-unverified-image:containers-storage:${container}
+
+# Re-pack it as a (chunked) container
+skopeo inspect --raw oci:${p_tmpdir}/test.oci > p_manifest.json
+start=`date +%s.%N`
+/home/redhat/work/dev-updates-remoting/rpm-ostree/rpm-ostree compose container-encapsulate --compare-with-build=oci:/tmp/coreTest-${compare_ver}/test.oci --ocimeta="${tmpdir}/p_manifest.json" --repo=repo testref oci:${tmpdir}/test.oci
```

Comment:
This path won't work outside your laptop.  Personally what I do is `makesudoinstall` (or just `install rpm-ostree /usr/bin`) in my dev container.